docs/hooks/hook_civicrm_alterMailContent.md

```diff
@@ -15,7 +15,13 @@ tokenizing it.
 
 ## Details
 
--   $content - fields include: html, text, subject, groupName, valueName, messageTemplateID
+$content - fields include: html, text, subject, groupName, valueName, messageTemplateID, mailingID
+Note that this hook is fired when: 
+- creating mailings through the traditional BAO mailer (standard CiviMail)
```

Comment:
```suggestion

* creating mailings through the traditional BAO mailer (standard CiviMail)
```
Bulleted lists should use the `*` syntax for the bullet and require a blank line above otherwise they won't render as a list.